### ISSUE TITLE
[Bug] Uncaught ReferenceError: require is not defined

### ISSUE THREAD

**Issue by J-F-Liu** [nozzle/react-static#1164]
After upgrade to latest react-static and related packages, yarn build is ok, but yarn start shows an error in browser console:
```
nodejsCustomInspectSymbol.mjs:16 Uncaught ReferenceError: require is not defined
    at register (nodejsCustomInspectSymbol.mjs:16)
    at Module.../../graphql/jsutils/nodejsCustomInspectSymbol.mjs (nodejsCustomInspectSymbol.mjs:50)
    at __webpack_require__ (bootstrap:723)
    at fn (bootstrap:100)
    at Module.../../graphql/jsutils/inspect.mjs (inspect.mjs:1)
    at __webpack_require__ (bootstrap:723)
    at fn (bootstrap:100)
    at Module.../../graphql/language/parser.mjs (blockString.mjs:134)
    at __webpack_require__ (bootstrap:723)
    at fn (bootstrap:100)
```

**Comment by huygn** [comment]
Having the same issue right now

**Comment by mtiger2k** [comment]
same issue here.

**Comment by dalisoft** [comment]
same here :)

**Comment by dalisoft** [comment]
Very strange, i am configured [react-static](https://github.com/nozzle/react-static) project from [template](https://github.com/nozzle/react-static/tree/master/packages/react-static/templates/basic) and [configured](https://github.com/dalisoft/react-static-dev-env), it worked, published to github. Now restarted to make sure everything is OK, oops. `require` is not defined. Added just `sass` and `css modules` plugin (then removed to check, same error).

**Comment by dalisoft** [comment]
I can confirm the bug fixed after [adding this lines](https://github.com/gaearon/react-hot-loader/pull/1250/files#diff-4c5fd1e2b1cc9a0c0d638400610e3457R7)

**Comment by dalisoft** [comment]
Here another solution.

static-config.js
```js
export default {
target: 'node'
}
```

**Note**: _This solution only for development and temporaily_

**Comment by theKashey** [comment]
If this is an FE error - could anybody add a breakpoint at the problem place and paste a whole module here.
It might be a case, that there is no `require`, but `__webpack_require_`.

Another idea - _later_ that code would call `reactHotLoader.register`. I am quite interested - would this function even been called, ie _require sometimes would exists_ . 

To fix the problem we have to understand the problem.

**Comment by J-F-Liu** [comment]
Issue is fixed after react-hot-loader released v4.8.5.

**Comment by dalisoft** [comment]
Great. Thanks to everyone who helped to fix this issue, @theKashey thanks :)
